include/LuaContext.hpp

```diff
@@ -1310,7 +1310,7 @@ class LuaContext {
             RealReturnType;
         
         // we push the parameters on the stack
-        auto inArguments = Pusher<std::tuple<TParameters...>>::push(state, std::make_tuple(std::forward<TParameters>(input)...));
+        auto inArguments = Pusher<std::tuple<TParameters...>>::push(state, std::forward_as_tuple((input)...));
```

Comment:
This breaks passing `unique_ptr` arguments, as `forward_as_tuple`, despite its name, does NOT actually forward; instead it would "forward" the `unique_ptr` as `unique_ptr<> const&`, which cannot be pushed.